components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -264,6 +264,16 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
         }.getAndConsumeRustString()
     }
 
+    /**
+     * New thing
+     */
+    fun authorizeAuthCode(clientId: String, scopes: Array<String>): String {
```

Comment:
naming nit: desktop calls this "authorizedOAuthCode", is it worth trying to be consistent between the two?

Reply:
Ugh, I'm sorry, I typod this as past-tense "authorized" where it should be the verb "authorize" :-(

Reply:
The distinction I was trying to make was "Auth" vs "OAuth", with desktop using the later.